test/fixtures/copy-parameters.css

```diff
@@ -1,5 +1,5 @@
 body {
-  clip-path: url("imported/pixel.png?#iefix")
+  clip-path: url("imported/pixel.png?foo=bar")
```

Comment:
could you add, not replace here (to ensure we do not change the current behavior) if this make sense?